app/templates/components/public/social-links.hbs

```diff
@@ -1,4 +1,7 @@
 <div class="ui list">
+  <div style="margin-bottom:4px;"><li>{{header-date @eventtime}} {{@eventtimezone}} To <span>{{header-date @eventtimeends}} {{@eventtimezone}}</span></li>
+  <a href="https://calendar.google.com/calendar/u/0/r/eventedit?dates={{gmt-time @eventtime}}/{{gmt-time @eventtimeends}}&text={{@eventname}}&location={{@eventlocation}}&sf=true">Add To Calender</a>
```

Comment:
No need of creating a new helper. Just create a computed property in the controller which returns this URL

Reply:
ohk